⚡ Global improvements in the code and some fixes (#161)
Androz2091/discord-giveaways: README.md

After that, giveaways that are not yet completed will start to be updated again and new giveaways can be started.
You can pass an options object to customize the giveaways. Here is a list of them:

-   **client**: the discord client (your discord bot instance).
-   **options.storage**: the json file that will be used to store giveaways.
-   **options.updateCountdownEvery**: the number of milliseconds it will take to update the timers.

> -   **options.endedGiveawaysLifetime**: the number of milliseconds that giveaways have been ended after which they can get deleted from the DB.

Reviewer: ```suggestion
-   **options.endedGiveawaysLifetime**: duration for which the ended giveaways remain in the database after they are ended.
```
Maybe this is clearer?

**options.newWinnerCount**: the new number of winners.  
**options.newPrize**: the new prize.  
**options.addTime**: the number of milliseconds to add to the giveaway duration.  

> **options.setEndTimestamp**: the timestamp of the new end date. `Date.now() + 1000`.

Reviewer: ```suggestion
**options.setEndTimestamp**: the timestamp of the new end date. (for example, for the giveaway to be ended in 1 hour, set it to `Date.now() + 60000`).
```
btw, we could change this to the following? Is it clearer?
